Fingers/GrooveDialog.h

```diff
@@ -6,6 +6,7 @@ class GrooveDialog : public SWS_DockWnd
 public:
 	GrooveDialog();
 	void Refresh() { if (IsValidWindow()) RefreshGrooveList(); }
+	void ApplySelectedGroove_public(); // NF: so we can call from action list

```

Comment:
Couldn't `ApplySelectedGroove` be made public directly?

Reply:
Thanks for the review.
> Couldn't ApplySelectedGroove be made public directly?

That's what I had as original version indeed but then I thought maybe it's not that good style making the whole function public which was private originally, so I added a 'dedicated' public version instead.
But you're right of course, practically makes no different, I just wasn't sure about 'coding style'.

Reply:
Implemented suggested change in https://github.com/reaper-oss/sws/commit/60cdaa9d73974164c2d92d4649368abe8fab2b69